src/yunohost/hook.py

```diff
@@ -330,7 +331,11 @@ def hook_exec(path, args=None, raise_on_error=False, no_trace=False,
         cmd_script = path
 
     # Construct command to execute
-    command = ['sudo', '-n', '-u', 'admin', '-H', 'sh', '-c']
+    if user == "root":
+        command = ['sh', '-c']
```

Comment:
This means we always consider this python code is executed as root. Shouldn't we add a sudo here just in case ?